android/src/main/java/com/revenuecat/purchases/common/mappers/SkuDetailsMapper.kt

```diff
@@ -1,45 +1,8 @@
-package com.revenuecat.purchases.common
+package com.revenuecat.purchases.common.mappers
 
 import com.android.billingclient.api.SkuDetails
-import com.revenuecat.purchases.EntitlementInfo
-import com.revenuecat.purchases.EntitlementInfos
-import com.revenuecat.purchases.Offering
-import com.revenuecat.purchases.Offerings
-import com.revenuecat.purchases.Package
-import com.revenuecat.purchases.PurchaserInfo
-import com.revenuecat.purchases.util.Iso8601Utils
-import org.json.JSONArray
-import org.json.JSONObject
 import java.text.NumberFormat
 import java.util.Currency
-import java.util.Date
-
-fun EntitlementInfo.map(): Map<String, Any?> =
-    mapOf(
-        "identifier" to this.identifier,
-        "isActive" to this.isActive,
-        "willRenew" to this.willRenew,
-        "periodType" to this.periodType.name,
-        "latestPurchaseDateMillis" to this.latestPurchaseDate.toMillis(),
-        "latestPurchaseDate" to this.latestPurchaseDate.toIso8601(),
-        "originalPurchaseDateMillis" to this.originalPurchaseDate.toMillis(),
-        "originalPurchaseDate" to this.originalPurchaseDate.toIso8601(),
-        "expirationDateMillis" to this.expirationDate?.toMillis(),
-        "expirationDate" to this.expirationDate?.toIso8601(),
-        "store" to this.store.name,
-        "productIdentifier" to this.productIdentifier,
-        "isSandbox" to this.isSandbox,
-        "unsubscribeDetectedAt" to this.unsubscribeDetectedAt?.toIso8601(),
-        "unsubscribeDetectedAtMillis" to this.unsubscribeDetectedAt?.toMillis(),
-        "billingIssueDetectedAt" to this.billingIssueDetectedAt?.toIso8601(),
-        "billingIssueDetectedAtMillis" to this.billingIssueDetectedAt?.toMillis()
-    )
-
-fun EntitlementInfos.map(): Map<String, Any> =
-    mapOf(
-        "all" to this.all.asIterable().associate { it.key to it.value.map() },
-        "active" to this.active.asIterable().associate { it.key to it.value.map() }
-    )
```

Comment:
This file used to be mappers.kt and extracted a lot of the mappers to their own files. I moved the SkuDetails related functions to `SkuDetailsMapper.kt` but Git things that I renamed the `mappers.kt` file.